src/CMakeLists.txt

```diff
@@ -291,6 +291,9 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   set(LEANC_STATIC_LINKER_FLAGS "-lleancpp -lInit -lStd -lLean")
 elseif(${CMAKE_SYSTEM_NAME} MATCHES "Emscripten")
   set(LEANC_STATIC_LINKER_FLAGS "-lleancpp -lInit -lStd -lLean -lnodefs.js")
+elseif(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+  # `-pie` defaulting is not consistent on Linux distributions, so let's default to off
+  set(LEANC_STATIC_LINKER_FLAGS "-no-pie -Wl,--start-group -lleancpp -lInit -lStd -lLean -Wl,--end-group -lbcrypt")
```

Comment:
Let's add `-lbcrypt` to `LEANC_STATIC_LINKER_FLAGS` in a separate block so as not to duplicate this line

Reply:
And I guess it should be added to `LEANC_SHARED_LINKER_FLAGS` as well